M2/Macaulay2/m2/system.m2

```diff
@@ -144,10 +144,12 @@ exit = Command exitMethod
 
 setRandomSeed = method(Dispatch => Thing)
 setRandomSeed ZZ := seed -> (
+    if recursionDepth() >= 3 then
```

Comment:
Why is this 3 and not 1?

Reply:
Oh never mind, I figured it out!

Reply:
Idk, recursion depth is not precise when using methods.